config/routes.rb

```diff
@@ -95,10 +95,6 @@
     match '/:person_id/settings/payments/paypal_account/show' => 'paypal_accounts#show', :as => :show_paypal_account_settings_payment
     match '/:person_id/settings/payments/paypal_account/create' => 'paypal_accounts#create', :as => :create_paypal_account_settings_payment
 
-    scope module: "atom", constraints: { format: "atom" } do
```

Comment:
Should we do a redirect, so that URLs pointing to this route will not fail. My assumption is that there are a lot of users using exactly this route, since this is the "new" route.